sed/loader/flash/loader.py

```diff
@@ -650,16 +650,32 @@ def buffer_file_handler(self, data_parquet_dir: Path, detector: str):
         parquet_filenames = [
             buffer_file_dir.joinpath(Path(file).stem + detector) for file in self.files
         ]
+        existing_parquet_filenames = [file for file in parquet_filenames if file.exists()]
 
         # Raise a value error if no data is available after the conversion
         if len(h5_filenames) == 0:
             raise ValueError("No data available. Probably failed reading all h5 files")
 
+        if not force_recreate:
+            # Check if the available channels match the schema of the existing parquet files
+            schemas = [pq.read_schema(file) for file in existing_parquet_filenames]
+            available_channels_set = set(self.available_channels)
+
+            for i, schema in enumerate(schemas):
+                schema_set = set(schema.names)
+                # Check if available_channels are the same as schema including pulseId
+                if schema_set != available_channels_set.union({"pulseId"}):
+                    raise ValueError(
```

Comment:
This will always rise, as there are columns generated by the loader which are not in the config.

I tested with parquets generated before this PR and with ones generated in this PR. Both rise every time

Reply:
I fixed this in a commit. I know its not good practice, but was easier to fix than to explain! :)

Reply:
It's maybe a matter of style, but why raise an error of they don't match? I would raise a warning, and recreate the files.

Reply:
if the config is wrong, you don't want to overwrite the parquets. So in my opinion the error is necessary, not only a matter of style